test/e2e/scheduling/limit_range.go

```diff
@@ -222,6 +226,89 @@ var _ = SIGDescribe("LimitRange", func() {
 		framework.ExpectNoError(err)
 	})
 
+	ginkgo.It("should ensure that a limitRange can be listed, patched and deleted by collection", func() {
```

Comment:
Does this has something to do with scheduling? It seems like we're testing limitRange itself.

Reply:
As part of addressing the [conformance technical debit](https://apisnoop.cncf.io/conformance-progress/endpoints/1.26.0/?filter=untested), the focus of this PR is to create an e2e test that will confirm the behaviour of the following limitRange endpoints.

- [listCoreV1LimitRangeForAllNamespaces](https://apisnoop.cncf.io/1.26.0/stable/core/listCoreV1LimitRangeForAllNamespaces?conformance-only=true)
- [patchCoreV1NamespacedLimitRange](https://apisnoop.cncf.io/1.26.0/stable/core/patchCoreV1NamespacedLimitRange?conformance-only=true)
- [deleteCoreV1CollectionNamespacedLimitRange](https://apisnoop.cncf.io/1.26.0/stable/core/deleteCoreV1CollectionNamespacedLimitRange?conformance-only=true)

The details of the reseach is captured in the issue [#112429](https://github.com/kubernetes/kubernetes/issues/112429).

Once the test proven stable (2 weeks flake free) we will promote it to conformance.

Reply:
Of course, these tests are helpful. My concern here is do these tests have more or less relevances with pod scheduling, I didn't find any. If so, maybe put them under the package only related to limit range will be better, for maintainabilities. I don't know if we have one.

Reply:
We placed the test in the limit_range.go file as it already contain the test for create -, delete -, list -, read -, replaceCoreV1NamespacedLimitRange endpoints, which is nested under test/e2e/scheduling/.
Moving the limit_range.go file to another location would likely have to be a SIG Scheduling decision and not related to the test.